docs/conf.py

```diff
@@ -134,37 +146,35 @@
 # [howto, manual, or own class]).
 latex_documents = [
     (master_doc, 'minos.tex',
-     'Minos Microservice Networks Documentation',
+     'Minos Microservice Netowrk Documentation',
```

Comment:
Fast typing my friend? 😏
```suggestion
     'Minos Microservice Network Documentation',
```